deepchem/models/losses.py

```diff
@@ -126,6 +126,16 @@ def __call__(self, output, labels):
     return tf.nn.sparse_softmax_cross_entropy_with_logits(labels, output)
 
 
+class NegLogLoss(Loss):
```

Comment:
Could you explain what this class is for?  I see several things that look like potential problems.  1. The name is confusing because it doesn't take a log.  2. It doesn't look like a loss function at all, because it completely ignores the labels.  It looks more like a regularizer?  3. Loss functions should produce a one or two dimensional (depending on the input) tensor as output with a loss for each sample or sample/task pair.  That gets multiplied by the weights from the dataset.  This only produces a single number.

Reply:
Since I'm using `tfp.distributions.TransformedDistribution`, there are already methods available to compute log likelihoods of samples and the Kullback-Leibler divergence between the learned distribution and the target distribution. I will work on implementing these losses in a way that is consistent with `dc.models.losses.Loss`.